recipe_scrapers/hellofresh.py

```diff
@@ -11,7 +14,21 @@ def title(self):
         return self.schema.title()
 
     def total_time(self):
-        return self.schema.total_time()
+        script_tag = self.soup.find("script", {"id": "__NEXT_DATA__"})
+
+        if script_tag:
+            script_content = script_tag.string
+            total_time_match = re.search(r'"totalTime":"(PT\d+M)"', script_content)
+            prep_time_match = re.search(r'"prepTime":"(PT\d+M)"', script_content)
+
+            if total_time_match and prep_time_match:
+                total_time_str = total_time_match.group(1)
+                prep_time_str = prep_time_match.group(1)
+
+                total_time_minutes = get_minutes(total_time_str)
+                prep_time_minutes = get_minutes(prep_time_str)
+
+                return total_time_minutes + prep_time_minutes
```

Comment:
Could you add a comment to explain that Hellofresh is a bit odd about using the `totalTime` metadata field to represent cook time, and then `prepTime` has to be added to that?

(also: any reason not to add the `cook_time` and `prep_time` methods to the scraper while we're doing this?)

Reply:
I can definitely rework it into two separate methods and than just have total_time return the sum of the two, I'll try that in just a minute.

Reply:
Done, let me know what you think!